packs/vsphere/actions/workflows/vm_hw_basic_build.yaml

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
I like this approach of VM creation i.e. using a workflow. It might end up being less efficient as every task is an API call against vsphere however there is immense clarity and pluggability which IMO trumps the cost of API calls.